modules/openapi-generator/src/test/java/org/openapitools/codegen/swift3/Swift3ModelTest.java

```diff
@@ -98,10 +98,10 @@ public void simpleModelTest() {
 
         final CodegenProperty property4 = cm.vars.get(3);
         Assert.assertEquals(property4.baseName, "binary");
-        Assert.assertEquals(property4.dataType, "URL");
+        Assert.assertEquals(property4.dataType, "Data");
```

Comment:
because changing https://github.com/OpenAPITools/openapi-generator/pull/995/files#r215861936, need to fix test.
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift3Codegen.java#L142
says that  "binary" returns "Date", so I thinks this change is reasonable.